packages/coin-base/src/derive/ed25519.ts

```
@@ -0,0 +1,111 @@
+  };
+}
+
+const isValidPath = (path: string): boolean => {
+  if (!pathRegex.test(path)) {
+    return false;
+  }
+  return !path
+    .split('/')
+    .slice(1)
+    .map(replaceDerive)
+    .some(isNaN as any /* ts T_T*/);
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)